### PR TITLE
fix: Fm 298 testcase and bug fixes

### PR DESCRIPTION
**Pull request by ashwinnair-chimple**
# Changes
- Changed the testcase spec file
- Fixed timer ticking issues when going to next screen and retry

# How to test
- Play any level and on completion select next or retry the time ticker should load

Ref: [FM-298](https://curiouslearning.atlassian.net/browse/FM-298)

[FM-298]: https://curiouslearning.atlassian.net/browse/FM-298?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bernhardccodev — APPROVED** [review]
Looks good to me.

**Review by woodsjohn-codev — APPROVED** [review]

File: src/components/timerHtml/timerHtml.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    id: string = 'timer-html-component',
+    options: BaseHtmlOptions = { selectors: TIMER_HTML_SELECTORS }
+  ) {
+    super(options, id, TIMER_HTML_LAYOUT);
+    // Automatically render the component on instantiation
+    this.render();
+
+  }
+
+  destroy() {
```
Comment (miguelccodev) [inline src/components/timerHtml/timerHtml.ts:30]: No need to override if there are no difference.

File: src/components/timer-ticking/timer-ticking.spec.ts
Comment (miguelccodev) [inline src/components/timer-ticking/timer-ticking.spec.ts:1]: Can we implement at least 1 test case?

**Review by miguelccodev — CHANGES_REQUESTED** [review]

File: src/components/timer-ticking/timer-ticking.spec.ts
Comment (ashwinnair-chimple) [reply src/components/timer-ticking/timer-ticking.spec.ts:1]: i have included test cases in timerHTML.spec.ts 

as timerHTML is inluded in timer-ticking.ts

File: src/components/timerHtml/timerHtml.ts (hunk 1 above)
Comment (ashwinnair-chimple) [reply src/components/timerHtml/timerHtml.ts:30]: Noted, Thanks

**Review by miguelccodev — APPROVED** [review]

**Pull request merged**